windows/src/desktop/inst/download.in

```diff
@@ -9,19 +9,16 @@ default:
 
 copyredist-desktop:
   -mkdir $(ROOT)\release\$VERSION
-  copy /Y keymandesktop100.msi $(ROOT)\release\$VERSION\keymandesktop100.msi
-  copy /Y keymandesktop100.exe $(ROOT)\release\$VERSION\keymandesktop-$VERSION.exe
+  copy /Y keymandesktop.msi $(ROOT)\release\$VERSION\keymandesktop.msi
+  copy /Y keymandesktop.exe $(ROOT)\release\$VERSION\keymandesktop-$VERSION.exe
```

Comment:
Isn't the target now `keymandesktop.exe`?

Reply:
No, we maintain the version number in the .exe, just remove in lower level files.